src/components/TableOfContentsSidebar.vue

```diff
@@ -81,28 +81,28 @@ export default {
     return {
       isActive: true,
       isOpen: -1,
-      accordian: {},
-      accordianList: [],
```

Comment:
I'm pretty sure this is effectively a local api. 

https://en.wikipedia.org/wiki/Accordion